File: backup_before_industrialisation_20250626/backend/components/metiers/ressources_humaines/utils/plugins/fallback/fallback.py
# fallback.py
# Fallback plugin Python pour Ressources_humaines – exemple clé en main


def plugin_fallback(name):
    """
    Fallback minimal : plugin de secours qui log l'appel
    """
    return f"[PLUGIN-FALLBACK] Appel du plugin: {name}"

File: backend/components/metiers/mobile/utils/plugins/fallback/fallback.js
// fallback.js
// Fallback plugin JS pour Mobile – exemple clé en main

/**
 * Fallback minimal : plugin de secours qui log l'appel
 * @param {string} name
 * @returns {string}
 */
function pluginFallback(name) {
  return `[PLUGIN-FALLBACK] Appel du plugin: ${name}`;
}

module.exports = { pluginFallback };
